services/ui-src/src/utils/errors/errorHandler.test.ts

```diff
@@ -1,30 +1,41 @@
 import { errorHandler } from "./errorHandler";
 
-jest.spyOn(window, "alert").mockImplementation(() => {});
+const mockCallback = jest.fn(() => {});
+const errorMessage = "Test error message";
+
+beforeEach(() => {
+  jest.spyOn(console, "log").mockImplementation(jest.fn());
```

Comment:
This keeps the `console.log` side-effects of `errorHandler` out of the `jest` console so it's less noisy.